bindings_generator/src/api.rs

```diff
@@ -583,3 +613,886 @@ impl Ty {
         }
     }
 }
+
+pub fn module_name_from_class_name(class_name: &str) -> String {
+    // There are a few cases where the conversions do not work:
+    // Node2D => node_2d vs Vec3Uniforhm => vec3_uniform
+    if class_name == "VisualShaderNodeVec3Uniform" {
+        return "visual_shader_node_vec3_uniform".into();
+    }
+    if class_name == "VisualShaderNodeVec3Constant" {
+        return "visual_shader_node_vec3_constant".into();
+    }
```

Comment:
I feel like those are very specific and this could "break" with custom bindings.

What about doing something like `class_name.replace("Vec3", "_vec3_")` later on instead? Could this work?

Reply:
We could forego these too. We are trading improvements for module names like `node_2d` (of which there are more).
It would possible to replace a token in the result `_vec_3` for `_vec3_`. I doubt there would be false matches.